packages/lab/src/calendar/internal/CalendarDay.css

```diff
@@ -63,106 +57,105 @@
 }
 
 .uitkCalendarDay:focus-visible {
-  outline: var(--calendar-day-focused-outline);
+  outline: var(--calendarDay-focused-outline);
   outline-offset: calc(0px - var(--uitk-focused-outlineWidth));
 }
 
 .uitkCalendarDay-outOfRange {
-  color: var(--calendar-day-out-of-range-text-color);
+  color: var(--calendarDay-outOfRange-text-color);
 }
 
 .uitkCalendarDay:hover {
-  background: var(--calendar-day-background-hover);
-  color: var(--calendar-day-text-color-hover);
+  background: var(--calendarDay-background-hover);
+  color: var(--calendarDay-text-color-hover);
 }
 
-.uitkCalendarDay-unselectableHighEmphasis,
-.uitkCalendarDay-unselectableHighEmphasis:hover {
-  color: var(--calendar-day-blocked-text-color);
-  cursor: var(--calendar-day-blocked-cursor);
-  background: var(--calendar-day-blocked-background);
+.uitkCalendarDay-unselectable.uitkMediumEmphasis,
```

Comment:
we use uitkEmphasisMedium in other components (switch medium / emphasis)

Reply:
also im curious, why is this an emphasis concept? as we usually have emphasis coming from characteristics and this is component specific

Reply:
@bhoppers2008 would probably be better off answering this